### PR TITLE
docs: reuse the pyproject metadata in sphinx docs

### PR DESCRIPTION
**Pull request by sennierer**

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -66,17 +71,17 @@
 
 # General information about the project.
 project = "APIS"
-copyright = "2021, Matthias Schlögl"
-author = "Matthias Schlögl"
+author = ", ".join([x.split("<")[0].strip() for x in pyproject["authors"]])
+copyright = f"{datetime.now().year}, {author}"
```
Comment (b1rger) [inline docs/source/conf.py:75]:
```suggestion
copyright = f"2016-{datetime.now().year}, {author}"
```

File: docs/source/conf.py [hunk 2]
```diff
@@ -93,6 +98,8 @@
 # The name of the Pygments (syntax highlighting) style to use.
 pygments_style = "sphinx"
 
+myst_gfm_only = True  # Enable GitHub flavored markdown
```
Comment (b1rger) [inline docs/source/conf.py:101]: This leads to
```
ModuleNotFoundError: Linkify enabled but not installed.
```

**Review by b1rger — CHANGES_REQUESTED** [review]

File: docs/source/conf.py (hunk 2 above)
Comment (sennierer) [reply docs/source/conf.py:101]: oh yes, needs to be added to pyproject.toml

**Review by b1rger — APPROVED** [review]
(please remove the added empty line when squashing)

File: docs/source/conf.py (hunk 2 above)
Comment (sennierer) [reply docs/source/conf.py]: removed the GitHub flavored markdown

**Review by b1rger — APPROVED** [review]

**Pull request merged**